src/test/resources/automation.xml

```diff
@@ -227,10 +227,21 @@
         <!--<className>org.wso2.carbon.automation.extensions.servers.carbonserver.CarbonServerExtension</className>-->
         <platformExecutionManager>
             <extentionClasses>
+                <class>
+                    <name>org.wso2.connector.integration.test.base.CopyConnectorZipToResources</name>
+                </class>
                 <class>
                     <name>org.wso2.esb.integration.common.extensions.carbonserver.CarbonServerExtension</name>
                     <parameter name="-DportOffset" value="200" />
+                    <!--For EI-->
```

Comment:
Shall we say

<!-- If you are running tests against WSO2 EI 6.x, uncomment below--> 
<parameter name="managementPort" value="9763" />
<parameter name="startupScript" value="integrator" />

<!-- If you are running tests against WSO2 EI 7.x, uncomment below-->
<!--<parameter name="startupScript" value="micro-integrator" />-->

Reply:
can't we add a profile for EI or MI and set these as system properties when we run mvn clean install -P profile .. 
Then only the execution command will change , and we don't need to edit the files.